content/post/cheers-to-small-beginnings.md

```diff
@@ -0,0 +1,55 @@
+
+The visual translation started fuelled with qualitative data, quantitative data and our company culture. We first started looking for some typography options, and in the process, we stumble upon [Free Font Ballon](https://www.behance.net/gallery/55482949/Free-Font_Balloon) by the incredibly talented [Aga Magdziak](https://www.behance.net/aga735d). After a couple of minutes we found out that there was more to Aga than meets the eye. Along with the typeface, Aga created a huge set of visual artifacts that caught our attention. We like to think of what we discovered as open source design, and we deeply believe in it. In the end we came up with a design lexicon - a branch of knowledge - that empowered us create the Brand Persona, and it was data-oriented all the way.
+
+![{d}ali Lexicon](/post/cheers-to-small-beginnings/{d}ali-lexicon.png)
+
+### Claws that can grab a dinosaur
+
+With a lexicon, we had all we needed to create the Brand Persona’s visual translation. After vivid care on content analysis we chose to give birth to {d}ali, a quite particular parrot. A parrot has the distinct characteristic of easily learn languages, and that pretty much stroked our agenda. Then, we had to make sure that our Brand Persona personality and traits would reflect on {d}ali aesthetics. Its claws translate our brand’s trustworthiness, its tail translate our brand’s enthusiasm and {d}ali’s head is all about being genuine, mainly because it gathers characteristics from several parrots, each one reinforcing our rational. And this, is how {d}ali was born.
+
+![{d}ali](/post/cheers-to-small-beginnings/{d}ali.png)
+
+We deeply believe in co-creation. Our pathway towards innovation is not to simply share our evolution, but rather to evolve together with the community one step at a time.
+
+We have an honest attention to the group, along with an extreme care on the individual. Like as in good typography, we care about our community not as a group of important people, but an important group of people, by choice.
+
+So please, feel free to check the assets we’ve created for the Brand Persona and be so kind to share your thoughts on our brand new design channel in our [Slack Community](http://missing-link)
+
+Here you can find the [source{d}'s Brand Persona](http://missing-link) and [{d}ali's Lexicon](http://missing-link).
+
+### One, two, buckle my shoe
+
+This is base one, and we’ve reached it. Now, we’re studying our brand touchpoints that will allow us to give birth to a design system and to reach second base. In order to move to third base, we need to reflect our brand unique values and personality traits on the connection between its every single community touchpoints. Defining how the company, as a person, builds its brand relationship with our community is the fourth base. So, stay tuned, more news will come soon(ish)!
```

Comment:
I think "base one" is really "square one" or "first milestone" or whatever. I have never read "base one".

Reply:
Now ~,~ we’re

Reply:
give birth to -> produce

Reply:
to reach **the** second base

Reply:
move to **the** third base

Reply:
its brand relationship -> it's brand relationship

Reply:
Thanks! All changed, except.

> its brand relationship

The brand relationship. Its own.

> it's brand relationship

it is brand relationship.